colin-api/src/colin_api/models/filing.py

```diff
@@ -226,94 +235,104 @@ def _create_filing(cls, cursor, event_id: str,  # pylint: disable=too-many-argum
                     TO_TIMESTAMP_TZ(:effective_dt,'YYYY-MM-DD"T"HH24:MI:SS.FFTZH:TZM')
                 """
             )
-            if filing_type_code in ['OTANN', 'ANNBC']:
+            filing_type_code = filing.get_filing_type_code()
+            if filing_type_code in ['OTANN']:
                 insert_stmnt = insert_stmnt + ', period_end_dt, agm_date, arrangement_ind, ods_typ_cd) '
                 values_stmnt = values_stmnt + \
                     ", TO_DATE(:period_end_date, 'YYYY-mm-dd'), TO_DATE(:agm_date, 'YYYY-mm-dd'), 'N', 'P')"
                 cursor.execute(
                     insert_stmnt + values_stmnt,
-                    event_id=event_id,
-                    filing_type_code=filing_type_code,
-                    effective_dt=effective_date,
+                    event_id=filing.event_id,
+                    filing_type_code=filing.get_filing_type_code(),
+                    effective_dt=filing.effective_date,
                     period_end_date=ar_date if not agm_date else agm_date,
                     agm_date=agm_date
                 )
-            elif filing_type_code in ['OTADD', 'NOCAD', 'OTCDR', 'NOCDR', 'OTINC', 'BEINC']:
-                insert_stmnt = insert_stmnt + ', period_end_dt) '
-                values_stmnt = values_stmnt + ", TO_DATE(:period_end_date, 'YYYY-mm-dd'))"
+            elif filing_type_code in ['OTADD', 'OTCDR', 'OTINC']:
```

Comment:
Do you have to include the new fee code here BCADD, BCCDR etc.?

Reply:
No these are 'filing type codes' that are only used in colin

Reply:
We don't use these in our lear db at all -> in lear db we use 'changeOfAddress', etc. and no specific filing type code. The fee codes are different (we don't send those back to colin)